pyright/master/requirements-pinned.txt

```diff
@@ -375,18 +378,6 @@ noteable-origami==1.1.5
 notebook==7.2.2
 notebook-shim==0.2.4
 numpy==1.26.4
-nvidia-cublas-cu12==12.1.3.1
```

Comment:
I wish there was a way we could stop these pins from getting set based on whatever someone has installed locally vs what we actually need for the dagster packages we use - these nvidia deps keep causing problems for me